### DISCUSSION TITLE
JPEG Output Quality

### DISCUSSION THREAD

**Discussion by ConfusedPiratesSociety** [nftchef/art-engine#139] [Help + Q&A]
Hey,

I've had a good look but cannot find how to affect the JPEG output quality. I'm trying to use some tiny text which outputs fine on png but is unreadable on the jpg output. But the png filesize is too big. I know you're going to say that is the price of compression but I would like to investigate the setting all the same.

Is this possible?

Cheers,

CP

**Comment by nftchef** [comment]
Are the file dimensions same as the source files, or is there any resizing going on?

**Reply by dakky221** [reply]
Don't want to open another issue, but I'm not resizing anything and yet JPEGs clearly show signs of compression.
Is there a way to put 85% or 90% quality instead of default 75%? 

75% makes files around 65KB, while 85% would make them still well under 100KB which is acceptable for 10k collection... and reduce the compression visibility...

**Reply by dakky221** [reply]
adding Quality in main.js seems to solves the problem

`canvas.toBuffer(`${outputJPEG ? "image/jpeg" : "image/png"}`, { quality: 0.9})`
